trading/trader/trader.py

```diff
@@ -363,8 +363,7 @@ def create_order_instance(self, order_type, symbol, current_price, quantity,
             # debug purpose
             profitability_str = PrettyPrinter.portfolio_profitability_pretty_print(profitability,
                                                                                    profitability_percent,
-                                                                                   self.get_trades_manager().
```

Comment:
The `trades_manager` parameter was only used to get the `reference_market` ?

Reply:
self.get_trades_manager() is also used for self.get_trades_manager().get_profitability(), etc